src/box/iproto.cc

```diff
@@ -1178,7 +1262,9 @@ iproto_connection_delete(struct iproto_connection *con)
 	       con->obuf[0].iov[0].iov_base == NULL);
 	assert(con->obuf[1].pos == 0 &&
 	       con->obuf[1].iov[0].iov_base == NULL);
+	struct iproto_thread *iproto_thread = con->iproto_thread;
 	mempool_free(&con->iproto_thread->iproto_connection_pool, con);
+	check_no_active_connections(iproto_thread);
```

Comment:
con->iproto_thread and you don't need extra `iproto_thread` variable

Reply:
`con` is freed here